lib/gds_api/json_client.rb

```diff
@@ -242,20 +239,9 @@ def do_request(method, url, params = nil, additional_headers = {})
         url: url,
       }
 
-      case method
-      when :get, :delete
```

Comment:
Minor: I found this commit a bit hard to follow. I think it might help to explain why you no longer need this condition and how come all the tests still pass.

Reply:
I've updated the commit message in an attempt to better explain the change.

Reply:
Looks great! 👍